requirements.txt

```diff
@@ -1,4 +1,4 @@
 django>=1.4.12
-PyJWT>=0.2.1
+PyJWT==0.2.1
```

Comment:
@clintonb In order to make the tests pass, I had to pin the version number for PyJWT to 0.2.1, since the unit tests fail with the latest version of that library.  @rlucioni mentioned you may have further insight about that library.  Can you confirm that this change (pinning the version number) is valid and makes sense for this repo?  Thanks.